CRM/Core/Smarty/plugins/function.help.php

```diff
@@ -67,6 +67,12 @@ function smarty_function_help($params, &$smarty) {
     $name = trim(strip_tags($params['title']));
   }
 
+  if (empty($name)) {
```

Comment:
If you have an extra.hlp file (usage described at https://github.com/civicrm/civicrm-core/pull/13488), the body part still works but for the title it ALWAYS overrides whether you put override=1 or not. I'm not sure how often titles are appended to rather than overwritten so probably not be a blocker.

Reply:
Easy enough to fix, have done. Also added a small bonus fix to the description above (or can break that out into a separate PR if preferred).